app/rental_routes.py

```diff
@@ -0,0 +1,143 @@
+import re
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from flask import Blueprint, request, jsonify, make_response
+from app import db
+from datetime import date, timedelta
+from dotenv import load_dotenv
+
+load_dotenv()
+
+rental_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+customer_rental_bp=Blueprint("customer_rental",__name__, url_prefix="/customers")    
+video_rental_bp = Blueprint("video_rentals", __name__, url_prefix="/videos")
+
+# POST /rentals/check-out
+@rental_bp.route("/check-out", methods=["POST"])  # strict_slashes=False??
+def rental_check_out():
+    request_body = request.get_json()
+
+    if "customer_id" not in request_body or "video_id" not in request_body:
+        return {
+            "details": f"Not found"
+        }, 400
+
+    if not isinstance(request_body["customer_id"], int):
+        return {
+            "details": f"Not found"
+        }, 400
+
+    customer = Customer.query.get(request_body["customer_id"])
+    if customer == None:
+        return {
+            "details": f"Not found"
+        }, 400
```

Comment:
You can use `get_or_404` to do this more cleanly:

```suggestion
    customer = Customer.query.get_or_404(request_body["customer_id"])
```